run/service-auth/receive_auth_requests_test.py

```diff
@@ -158,19 +156,15 @@ def test_authentication_on_cloud_run(
 
     assert response.status_code == HTTPStatus.OK
     assert "Hello" in response_content
-    assert "anonymous" not in response_content
 
 
-def test_anonymous_request_on_cloud_run(client: Session, endpoint_url: str) -> None:
+def test_anonymous_request_on_cloud_run_service(client: Session, endpoint_url: str) -> None:
     response = client.get(endpoint_url)
-    response_content = response.content.decode("utf-8")
 
-    assert response.status_code == HTTPStatus.OK
-    assert "Hello" in response_content
-    assert "anonymous" in response_content
+    assert response.status_code == HTTPStatus.UNAUTHORIZED
 
 
-def test_invalid_token(client: Session, endpoint_url: str) -> None:
+def test_an_invalid_token_on_cloud_run_service(client: Session, endpoint_url: str) -> None:
```

Comment:
Tests don't typically need to be named like this, `test_invalid_token` describes the same test.